pep-0999.rst

```diff
@@ -0,0 +1,175 @@
+PEP: 999
```

Comment:
> PEP 12 also says we can use rst already…

That's my fault. I thought I can finish my work by now, but due to some personal issues I couldn't find the time to work on it yet.